### ISSUE TITLE
Crashes at comparison of two equal nodes

### ISSUE THREAD

**Issue by adius** [tmpvar/jsdom#1264]
`nodeA.isEqualNode(nodeB)` throws following error:

```
Uncaught TypeError: b._attributes.$getNode is not a function
      at nodeEquals (/Users/adrian/Projects/shaven/node_modules/jsdom/lib/jsdom/living/node.js:126:35)
      at core.Node.isEqualNode (/Users/adrian/Projects/shaven/node_modules/jsdom/lib/jsdom/living/node.js:168:12)
      at Object.done (/Users/adrian/Projects/shaven/test/index.js:219:15)
      at /Users/adrian/Projects/shaven/node_modules/jsdom/lib/jsdom.js:259:18
      at doNTCallback0 (node.js:417:9)
      at process._tickCallback (node.js:346:13)
```

**Comment by domenic** [comment]
What jsdom and Node.js versions?

**Comment by adius** [comment]
jsdom: 6.5.1
node: 4.2.1

**Comment by domenic** [comment]
I am sorry I did not get to this in 7.0.0 today. I hope I can do so quite soon (tomorrow?). It looks like the isEqualNode code just failed to update itself to use the new attributes.js file. But it wasn't covered by any tests (!?) so we never caught that. It shouldn't be very hard.

**Comment by domenic** [comment]
Released in 7.0.1, with actual test coverage! Thanks for finding this!!

**Comment by adius** [comment]
Great! Thanks a lot for implementing it!
